PRレビューのworkflowで、bot操作でもレビューが動くようにしたい

https://github.com/taktamur/tree-outliner-sync/actions/runs/21771887010/job/62820960012?pr=15で、以下のエラーが出ていた。
Run bun run ${GITHUB_ACTION_PATH}/src/entrypoints/run.ts
Auto-detected mode: agent for event: pull_request
Requesting OIDC token...
Attempt 1 of 3...
OIDC token successfully obtained
Exchanging OIDC token for app token...
Attempt 1 of 3...
App token successfully obtained
Using GITHUB_TOKEN from OIDC
Checking permissions for actor: claude[bot]
Actor is a GitHub App: claude[bot]
Mode: agent
Context prompt: /code-review:code-review taktamur/tree-outliner-sync/pull/15
Trigger result: true
Preparing with mode: agent for event: pull_request
Actor type: Bot
Error: Action failed with error: Workflow initiated by non-human actor: claude (type: Bot). Add bot to allowed_bots list or use '*' to allow all bots.
Error: Process completed with exit code 1.
Run curl -L \

ぱっと見、「botがpushしたから止まった」ように見える。これを「botがpushしてもPRレビューを行う」ようにしたい。
@claude 検討して

--- a/.github/workflows/claude-code-review.yml
+++ b/.github/workflows/claude-code-review.yml
@@ -39,6 +39,7 @@ jobs:
           plugin_marketplaces: 'https://github.com/anthropics/claude-code.git'
           plugins: 'code-review@claude-code-plugins'
           prompt: '/code-review:code-review ${{ github.repository }}/pull/${{ github.event.pull_request.number }}'
+          allowed_bots: 'claude[bot]'
           # See https://github.com/anthropics/claude-code-action/blob/main/docs/usage.md
           # or https://code.claude.com/docs/en/cli-reference for available options
 

